localgov_microsites_group.module

```diff
@@ -40,3 +42,35 @@ function localgov_microsites_menu_local_actions_alter(array &$local_actions) {
     $local_actions['group_content.group_node_add_page']['appears_on'][] = 'view.group_nodes.microsites_page';
   }
 }
+
+/**
+ * Implements hook_menu_locol_tasks_alter().
+ */
+function localgov_microsites_menu_local_tasks_alter(&$data, $route_name, RefinableCacheableDependencyInterface &$cacheability) {
+  if (isset($data['tabs'][0]['group.view'])) {
+    if (($group = \Drupal::routeMatch()->getParameter('group')) &&
```

Comment:
Hmm... this isn't going to work with https://github.com/localgovdrupal/localgov_microsites_group/commit/5b881e96ac64218ff3e1773505964400ebc13393#diff-31a6feeb998ebe17cbedc629279f120b16a3db15468702d0ca85b21e486654ddR114=

Or rather we're going to have to make it so that we can call this hook, and pass in the route it's acting on.